proto/storage/cluster.proto

```diff
@@ -280,15 +280,17 @@ message ClusterHealthStatus {
         HEALTHY       = 4;
     }
 
-    CollectorHealthInfo       collector_health_info             = 1;
+    CollectorHealthInfo        collector_health_info            = 1;
     AdmissionControlHealthInfo admission_control_health_info    = 8;
+    LocalScannerHealthInfo     local_scanner_health_info        = 10;
 
     // The following _health_status fields provide aggregated health status of the respective components and are assigned by central.
     HealthStatusLabel         sensor_health_status              = 2 [(gogoproto.moretags) = "search:\"Sensor Status,store\""];
     HealthStatusLabel         collector_health_status           = 3 [(gogoproto.moretags) = "search:\"Collector Status,store\""];
 
     HealthStatusLabel         overall_health_status             = 4 [(gogoproto.moretags) = "search:\"Cluster Status,store\""];
     HealthStatusLabel         admission_control_health_status   = 7 [(gogoproto.moretags) = "search:\"Admission Control Status,store\""];
+    HealthStatusLabel         local_scanner_health_status       = 11 [(gogoproto.moretags) = "search:\"Local Scanner Status,store\""];
```

Comment:
can we add a comment to the top of this `message` to say which tag is next since the tags are out of order?